src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.html

```diff
@@ -1,9 +1,8 @@
 <div [ngGrid]="gridConfig" (onDragStop)="updateModel()" (onResizeStop)="updateModel()">
```

Comment:
Ooops text is too small, just in case:

1. Container selector should be aligned to the Sidebar title
2. Should be 16px, just like in the Sidebar title